components/auto_segmentation/README.md

```diff
@@ -0,0 +1,39 @@
+## [Keras+Tensorflow Mask R-CNN Segmentation](https://github.com/matterport/Mask_RCNN)
+
+### What is it?
+* This application allows you automatically to segment many various objects on images.
```

Comment:
@MalekMFS , for markdown we use '-' instead of '*'. Could you please look at codacy complains and fix?

Reply:
I think there is a bug with Codacy. it force me to enter three spaces after `-` and before text, then it want to remove one space again and again.
If you check Codacy issues you see it has two spaces and it asks to remove another space.
I don't know how to play with Codacy without junk commits!